int32 id
string name
float32 humidity
flaot32 temperature
